spec/namespaces/transforms.yaml

```diff
@@ -0,0 +1,264 @@
+openapi: 3.1.0
+info:
+  title: OpenSearch Index Transforms API
+  description: OpenSearch Index Transforms API
+  version: 1.0.0
+paths:
+  /_plugins/_transform:
+    get:
+      operationId: transforms.get_transforms.0
+      x-operation-group: transforms.get_transforms
```

Comment:
`transform.get` is better since this action is in the context of the `transforms` namespace. Similarly we have `indices.create` and `indices.delete` instead of `indices.create_index` and `indices.delete_index`